.circleci/config.yml

```diff
@@ -180,38 +187,38 @@ workflows:
           requires:
             - install_gems
       - plugin_unit_test:
-          name: unit_test_analytics
-          path: Analytics
-          workspace: AnalyticsCategoryPlugin
-          scheme: AWSPinpointAnalyticsPlugin
+          name: unit_test_auth
+          path: Auth
+          workspace: AWSAuthPlugin
+          scheme: AWSAuthPlugin
```

Comment:
AWSCognitoAuthPlugin

Reply:
Thanks for catching--git didn't remove the old files when you changed the names, and I didn't do a Pod update to generate the new workspace files